Marlin/pins_ANET_10.h

```diff
@@ -21,84 +21,72 @@
  */
 
 /**
- * Sanguinololu board pin assignments
+ * Anet board pin assignments
  */
 
 #define BOARD_NAME "Anet"
 
-#if defined(__AVR_ATmega1284P__)
-#define LARGE_FLASH true
-#define VARIANT_STANDARD true
-#endif
-
-#define SANGUINOLOLU_V_1_2
-#include "pins_SANGUINOLOLU_11.h"
+#include "pins_SANGUINOLOLU_12.h"
```

Comment:
it is much easier just to list what is needed than to undef/def stuff here and there even for things which will never be connected to the anet board